sample-functions/WebhookStash/handler.go

```diff
@@ -3,25 +3,25 @@ package main
 import (
 	"fmt"
 	"io/ioutil"
+	"log"
 	"os"
 	"strconv"
 	"time"
-	"log"
 )
 
 func main() {
 	input, err := ioutil.ReadAll(os.Stdin)
 	if err != nil {
 		log.Fatalf("Cannot read input %s.\n", err)
-        return
+		return
 	}
 	now := time.Now()
 	stamp := strconv.FormatInt(now.UnixNano(), 10)
 
 	writeErr := ioutil.WriteFile(stamp+".txt", input, 0644)
 	if writeErr != nil {
 		log.Fatalf("Cannot write input %s.\n", err)
-        return
+		return
```

Comment:
Did you run `gofmt` on this source file? It may get rejected by Go build / CI if not.